src/workloads/workloads.csproj

```diff
@@ -0,0 +1,176 @@
+
+  <UsingTask TaskName="CreateLightCommandPackageDrop" AssemblyFile="$(MicrosoftDotNetBuildTasksInstallersTaskAssembly)" />
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+
+    <WorkloadIntermediateOutputPath>$(ArtifactsObjDir)workloads/</WorkloadIntermediateOutputPath>
+    <WorkloadOutputPath>$(ArtifactsBinDir)workloads/</WorkloadOutputPath>
+    <WorkloadOutputPath Condition="'$(workloadArtifactsPath)' != ''">$(workloadArtifactsPath)/</WorkloadOutputPath>
+    <PackageSource>$(ArtifactsShippingPackagesDir)</PackageSource>
+    <PackageSource Condition="'$(workloadPackagesPath)' != ''">$(workloadPackagesPath)/</PackageSource>
+  </PropertyGroup>
+
+  <!-- Arcade -->
+  <PropertyGroup>
+    <!-- Temp directory for light command layouts -->
+    <LightCommandObjDir>$(ArtifactsObjDir)/LightCommandPackages</LightCommandObjDir>
+    <!-- Directory for the zipped up light command package -->
+    <LightCommandPackagesDir>$(ArtifactsNonShippingPackagesDir)</LightCommandPackagesDir>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <WixToolsetPath>$(PkgWix)\tools</WixToolsetPath>
+    <SwixPluginPath>$(PkgMicroBuild_Plugins_SwixBuild_Dotnet)</SwixPluginPath>
+    <SwixBuildTargets>$(SwixPluginPath)\build\MicroBuild.Plugins.SwixBuild.targets</SwixBuildTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Workloads" Version="$(MicrosoftDotNetBuildTasksWorkloadsPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="WiX" Version="$(WixPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="MicroBuild.Plugins.SwixBuild.Dotnet" Version="$(SwixPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Installers" Version="$(MicrosoftDotNetBuildTasksInstallersPackageVersion)" GeneratePathProperty="True" />
+  </ItemGroup>
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+
+  <Target Name="Build" DependsOnTargets="GenerateVersions">
+    <ItemGroup>
+      <!-- Overrides for Visual Studio setup generation. If the workload definition IDs change,
+           these must be updated. -->
+      <ComponentResources Include="wasm-tools" Title=".NET WebAssembly Build Tools"
+                          Description="Build tools for WebAssembly ahead-of-time (AoT) compilation and native linking."/>
+      <ComponentResources Include="microsoft-net-runtime-android" Title=".NET Android Build Tools"
+                          Description="Build tools for Android compilation and native linking."/>
+      <ComponentResources Include="microsoft-net-runtime-android-aot" Title=".NET Android Build Tools (AoT)"
+                          Description="Build tools for Android ahead-of-time (AoT) compilation and native linking."/>
+
```

Comment:
Could we add `microsoft-net-runtime-ios` and `microsoft-net-runtime-maccatalyst` to this PR as well? I think that would fulfill all of the current xamarin/MAUI requirements.

Reply:
```
  microsoft-net-runtime-ios platforms does not support Windows and will be skipped (osx-arm64, osx-x64).
  microsoft-net-runtime-maccatalyst platforms does not support Windows and will be skipped (osx-arm64, osx-x64).
  microsoft-net-runtime-macos platforms does not support Windows and will be skipped (osx-arm64, osx-x64).
  microsoft-net-runtime-tvos platforms does not support Windows and will be skipped (osx-arm64, osx-x64).
  ```

No.

Reply:
what packages does windows need for them?

Reply:
For ios/tvos/catalyst, we will need to have the runtime packs be in a separate workload and installable on windows.

I am leaning towards having that work done in a quick follow up.

Reply:
We are currently trying to include the iOS and MacCatalyst (and perhaps eventually tvOS and macOS) workloads from xamarin-macios in VS on windows, and those extend `microsoft-net-runtime-ios` and `microsoft-net-runtime-maccatalyst`. The manifest generated from the iOS and MacCatalyst workloads currently lists these runtime workloads as dependencies, and VS won't build without them. Plans to try to support this in a separate PR sound good to me.

Reply:
https://github.com/dotnet/runtime/pull/55851